train.py

```diff
@@ -183,10 +183,11 @@ def lr_drop(epoch):
             optimizer.step()
 
         loss = losses.mean()
```

Comment:
`loss` should be renamed `mean_losses` or somesuch here.

Reply:
Updated